src/Chunk.ts

```diff
@@ -1125,4 +1126,37 @@ export default class Chunk {
 			}
 		);
 	}
+
+	private createGetPath(options: OutputOptions, inputBase: string) {
```

Comment:
As I see it, you do not have dynamic updates of `this.id` between the definition and usage of this function. Thus, it would be possible to provide the chunk id as an additional parameter and extract the whole method into a separate file in the `utils` folder. Which would also make it easier to extract some of the nested functions without cluttering everything up.